### DISCUSSION TITLE
`prefer_socket` doesn't work when connect OceanBase with prisma

### DISCUSSION THREAD

**Discussion by dengfuping** [prisma/prisma#25195] [Q&A]
### Question

The same issue is posted on https://github.com/prisma/prisma/issues/24010 and I supply two solutions. 

However, it's like a stone dropped into the sea and  no response at all. So I post it on Discussions to look forward  to receiving official support.

Thanks for any help!

**Comment by ludralph** [comment]
Hi @dengfuping 👋 

Thank you for creating the PRs.

This is just a quick note regarding community pull requests; We typically don't respond immediately, so it's perfectly normal if you don't hear back from us within a few days or weeks. We have an internal process where we review community PRs during our sprints and then assign them to team members for a thorough review. 

Please be rest assured that we will get to it in due course.

**Comment by dengfuping** [comment]
Hi @ludralph

Any progress or plan for issue https://github.com/prisma/prisma/issues/24010? It seems no update over the past several versions of `prisma`.

**Reply by ludralph** [reply]
Hi @dengfuping 

I don't have an update with regards that issue but you can see how we want to manage issue going forward [here](https://www.prisma.io/blog/prisma-orm-manifesto).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
